### PR TITLE
connected new GL, moved two json from Ingredient into Ingredients, moved readme.md in Grocery

### LINKED ISSUE #25: [Groceristar][connect new GL to Ultimate]
You should add data, related to a new lists into grocery.json

### LINKED ISSUE #35: [SearchForm][getMethods update]
replace allergy, cuisine, course, holidays, diets methods to attribute methods
So instead of having `getAllergy()` we'll have `getAttribute('allergy')`

Tell me if you'll need more details

### LINKED ISSUE #49: [chickenKyiv][nutritions]
Similar task to our previous work.
You should move json data from this folders
https://github.com/GroceriStar/static-data/tree/master/dist/data/simple/Nutritions
https://github.com/GroceriStar/static-data/tree/master/dist/data/simple/Nutritions2

- [ ] add methods to chickenkyiv.js
- [ ] add tests
- [ ] update readme

### PR DESCRIPTION
**Pull request by vadim9999**
Fixes #25 , fixes #35, fixes #49

### CODE REVIEW

**Comment by atherdon** [comment]
our tests are not passing - do you have an idea why? @vadim9999

**Comment by vadim9999** [comment]
@atherdon Somebody commented my code that I have wrote in search.js. I don't know why

**Comment by atherdon** [comment]
it was me, sorry, make it back please.
i was debuggin the issue that we have in our other projects.
and i'm working on migration from current(old) approach of plugins to a
modern ways.
but as it not finished - we'll continue our work on this version of plugin


On 30-Jul-2018 at 10:53 AM, Vadim wrote:


@atherdon <https://github.com/atherdon> Somebody commented my code that I
have wrote in search.js. I don't know why

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/GroceriStar/groceristar-fetch/pull/55#issuecomment-408824685>,
or mute the thread

File: search.js [hunk 1]
```diff
@@ -19,67 +19,67 @@ const zi = function(){
...
+	return parser( ingredients2 );
+}
+
+module.getMeasurements = function () {
+	return parser( measurements );
+}
+
+module.exports = module;
 
 module.exports = {
```
Comment (atherdon) [inline search.js]: please delete this statement

File: search.js [hunk 2]
```diff
@@ -19,67 +19,67 @@ const zi = function(){
   return 'faggot';
```
Comment (atherdon) [inline search.js]: please delete this function too

**Comment by atherdon** [comment]
will run tests on my local machine. Btw, did you add methods, related to nutritions to readme, @vadim9999

**Comment by vadim9999** [comment]
@atherdon I have added methods to readme.md

**Comment by atherdon** [comment]
@vadim9999 good. tests passed well on my local PC. publishing to npm right now

**Pull request merged**